src/ellip.jl

```diff
@@ -0,0 +1,210 @@
+using Base.Math: @horner, libm
+using Base.MPFR: ROUNDING_MODE
+"""
+Computes Complete Elliptic function of 1st kind.------- K(m)  ; 
```

Comment:
the convention is that docstrings begin with the calling convention, i.e.
```
"""
    elliptic1(x)

...
"""

Reply:
I have added the reference in the docs (index.md) already . Okay will add here too .